pkgs/development/tools/misc/kibana/default.nix

```diff
@@ -47,6 +47,14 @@ in stdenv.mkDerivation rec {
     sha256 = shas."${stdenv.hostPlatform.system}" or (throw "Unknown architecture");
   };
 
+  # Kibana specifies it specifically needs nodejs 8.11.4 but nodejs in nixpkgs is at 8.12.0.
+  # The <nixpkgs/nixos/tests/elk.nix> test succeeds with this newer version so lets just
+  # disable the version check.
+  postPatch = ''
+    sed -i 's|currentVersion === requiredVersion|true|' \
+      src/setup_node_env/node_version_validator.js
```

Comment:
I would prefer a patch here so we see when it no longer applies.

Reply:
Done.